core/modules/user/css/user.css

```diff
@@ -201,3 +201,46 @@ form:not(.user-admin-settings) .password-strength-help-text.strong em {
 .profile dd {
   margin: 0 0 1em 0;
 }
+.user-simplified-page .user-simplified-page-logo {
+  max-width: 150px;
+}
+/* Maintenance page styles */
+body.user-simplified-page-body {
+  padding: 1em;
+}
+.user-simplified-page-wrapper {
+  padding: 2em;
+  max-width: 770px;
+  min-width: 255px;
+  margin: 0 auto;
+  background: white;
+  position: relative;
+}
+@media all and (min-width: 768px) {
+  body.user-simplified-page-body  {
+    width: 100%;
+    display: table;
+  }
+  body.user-simplified-page-body  {
+    height: 100%;
+    box-sizing: border-box;
+  }
+  body.user-simplified-page-body .user-simplified-page {
```

Comment:
Is this CSS selector correct? I couldn't find a div with the class `.user-simplified-page`.

But we should try to use a unique class so we don't have to discern between `body` and `div` tags (or nested selectors).

Reply:
Its there though, the `table-cell` style needs layers of DIVs to work properly it seems.